pkg/asset/installconfig/powervs/client.go

```diff
@@ -138,12 +201,16 @@ func (c *Client) GetDNSZoneIDByName(ctx context.Context, name string) (string, e
 }
 
 // GetDNSZones returns all of the active DNS zones managed by CIS.
-func (c *Client) GetDNSZones(ctx context.Context) ([]DNSZoneResponse, error) {
+func (c *Client) GetDNSZones(ctx context.Context, publish types.PublishingStrategy) ([]DNSZoneResponse, error) {
 	_, cancel := context.WithTimeout(ctx, 1*time.Minute)
 	defer cancel()
 
 	options := c.controllerAPI.NewListResourceInstancesOptions()
-	options.SetResourceID(cisServiceID)
+	if publish == types.ExternalPublishingStrategy {
```

Comment:
same thing here (could this be someting else or nil?)

Reply:
Will change.

Reply:
Done